src/index.js

```diff
@@ -23,6 +23,8 @@ const postToSengrid = ({
     subject,
     reply_to: from,
     template_id: templateId,
+    categories: [category],
+    custom_args: JSON.stringify(uniqueArgs),
```

Comment:
Depending on your version of the API or how it is sent email, this field has a different name.

Reply:
https://docs.sendgrid.com/for-developers/sending-email/unique-arguments